### PR TITLE
Instructions and scripts to run with supervisor.

### PR DESCRIPTION
**Pull request by geonexus**
### REVIEWERS
@flopezag @jframos 

### DESCRIPTION
- Added instructions and scripts to run with supervisor.

### CODE REVIEW

**Comment by jframos** [comment]
LGTM

File: README.rst [hunk 1]
```diff
@@ -182,8 +182,50 @@ in that folder.
...
+
+::
+
+     sudo supervisorctl stop fiware-facts
+
+NOTE:
+Supervisor provides an “event listener” to subscribe to “event notifications”.
+The purpose of the event notification/subscription system is to provide a mechanism for
+arbitrary code to be run (e.g. send an email, make an HTTP request, etc)
+when some condition is met. That condition usually has to do with subprocess state.
```
Comment (flopezag) [inline README.rst]: I prefer

when some condition is satisfied

File: utils/facts_start [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+# File to execute fiware-facts component
+#
+# __author__ = 'gjp'
```
Comment (flopezag) [inline utils/facts_start]: it is not needed a comment here to define __author__. Isn't

File: utils/facts_start [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+NAME="fiware_facts"                                  # Name of the application
+SOCKFILE=/var/run/gunicorn.sock  # we will communicte using this unix socket
```
Comment (flopezag) [inline utils/facts_start]: Try to put the comment in the same tab.

communicte?

**Comment by flopezag** [comment]
LGTM

**Pull request merged**